### PR TITLE
Create text.css

### LINKED ISSUE #4367: Open event and create event text looks small in mobile view
**Is your feature request related to a problem? Please describe.**
when opening website in mobile the "create event" and "open event" text looks very small.

**Describe the solution you'd like**
a slight increase in font size of those text.

**Describe alternatives you've considered**
increase font size
**Additional context**
Screenshot_20200508-232701_2

can I work on this issue 
please let me know.

### PR DESCRIPTION
**Pull request by aroranipun04**
Fixes #4367

#### Short description of what this resolves:
It resolves the problem of less font size making it looker bigger in both desktops and mobile phones

#### Changes proposed in this pull request:

- text at the top navigation increased
- will look larger and thus resolves the issue

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [ ] My branch is up-to-date with the Upstream `development` branch.

### CODE REVIEW

File: text.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.topnav{
+      font size = 20px;
```
Comment (niranjan94) [inline text.css:2]: Codacy Issue found: [Expected COLON at line 2, col 12.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5496590)

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5496590&bid=18012297)

File: text.css (hunk 1 above)
Comment (aroranipun04) [reply text.css:2]: semicolon is used while ending a statement in css not colon

**Comment by snitin315** [comment]
Not a priority

**Pull request closed without merging**